.github/workflows/issue-regression-labeler.yml

```diff
@@ -0,0 +1,32 @@
+# Apply potential regression label on issues
+name: issue-regression-label
+on:
+  issues:
+    types: [opened, edited]
+jobs:
+  add-regression-label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+    - name: Fetch template body
+      id: check_regression
+      uses: actions/github-script@v7
```

Comment:
@nateprewitt I guess the commit corresponds to [v7.0.1](https://github.com/actions/github-script/commits/v7.0.1). Should we instead change it to `actions/github-script@v7.0.1`?

Reply:
Both `v7` and `v7.0.1` are the same commit. This may have been an error on their side.

The issue we're controlling for is supply chain attacks where someone gets access to a repo, deletes an existing tag, and replaces it pointing to another commit. All of our workflows use exact SHAs to prevent this and we should maintain that in any new workflow.

Reply:
If you're interested in more information, you can read the guidance for this specific issue [here](https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies). There are also a few resources describing the ossf scorecard at a high level if interested [[blog]](https://openssf.org/blog/2020/11/06/security-scorecards-for-open-source-projects/) [[readme]](https://github.com/ossf/scorecard/blob/main/README.md#what-is-scorecard).

Reply:
Done